.circleci/config.yml

```diff
@@ -5,9 +5,12 @@ jobs:
       - image: python:3.6
     steps:
       - checkout
+      - run:
+          name: Clone submodules
+          command: git submodule update --init --recursive
```

Comment:
Why did this work before?

Reply:
It didn't. I pushed the initial config directly to master.